TestHelper.psm1

```diff
@@ -326,7 +326,7 @@ function Initialize-TestEnvironment
         
     # Import the Module
     $Splat = @{
-        Path = $moduleRoot
+        Path = (Join-Path -Path $moduleRoot -ChildPath '\DscResourceTestHelper\')
```

Comment:
So this line is causing an issue with the recent changes to DscResource.Tests but I don't think this is the correct fix. I'm working on a solution